CodingChallenges/CC_157_Zoom_Annotations/2-final/index.html

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <script src="https://cdn.jsdelivr.net/npm/p5@1.1.9/lib/p5.min.js"></script>
+  <script src="https://unpkg.com/ml5@latest/dist/ml5.min.js" type="text/javascript"></script>
```

Comment:
```suggestion
  <script src="https://unpkg.com/ml5@0.5.0/dist/ml5.min.js" type="text/javascript"></script>
```

@violetcraze just noting that I prefer to link to a specific version of ml5 as changes in updated versions might break olde examples.